core/monitoring/user-events/src/main/scala/org/apache/openwhisk/core/monitoring/metrics/EventConsumer.scala

```diff
@@ -94,6 +95,9 @@ case class EventConsumer(settings: ConsumerSettings[String, String], recorders:
   private def processEvent(value: String): Unit = {
     EventMessage
       .parse(value)
+      .filter { e =>
+        !metricConfig.ignoredNamespaces.contains(e.namespace)
+      }
```

Comment:
Thinking about the ignore support again - May be we should still update the Global metrics so as to account for presence of such activations by ignored namespaces.

Further even having a namespace level metric (which drops the specific action name) would be helpful to still see overall aggregated metric for that namespace. It also helps to confirm if the namespace is not getting abused for some reason (say key leak!)

Reply:
Makes sense @chetanmeh. Added a new metric for total namespace activations in both `PrometheusRecorder` and `KamonRecorder`